tooling-e2e-tests/utils.sh

```
@@ -338,3 +356,25 @@ scan_and_assert_referrals() {
     log_progress "✅ All referrals are valid"
   fi
 } 
+
+####################################################################################################
+#### SCHEDULER #####################################################################################
+####################################################################################################
+
+start_scheduler_local() {
+  log_progress "🔄 Starting Scheduler"
+  cd ${ROOT_DIR}
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)